common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3308,4 +3323,13 @@ object CelebornConf extends Logging {
       .doc("the heartbeat interval between worker and client")
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("60s")
+
+  val TEST_ALTERNATIVE: OptionalConfigEntry[String] =
+    buildConf("celeborn.testAlternative.value")
+      .withAlternative("celeborn.testAlternative.replaceKey")
```

Comment:
`celeborn.test.` is a namespace.

```suggestion
    buildConf("celeborn.test.alternative.key")
      .withAlternative("celeborn.test.alternative.deprecatedKey")
```